caffe2/core/operator.h

```diff
@@ -550,7 +551,7 @@ class CAFFE2_API OperatorBase : public Observable<OperatorBase> {
   // Preferrably use c10::optional, but nvcc doesn't work
   std::unique_ptr<const c10::FunctionSchema> fn_schema_ = nullptr;
   vector<c10::IValue> ivalue_inputs_;
-  vector<c10::IValue*> ivalue_outputs_;
+  vector<at::Tensor> ivalue_outputs_;
```

Comment:
This looks very misnamed now

Reply:
Ok, naming them `newstyle_inputs_` and `newstyle_outputs_`. Open for suggestions if you have a better idea.